### ISSUE TITLE
Powered crossbows are too powered.

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#31126]
(WEB REPORT BY: saulmyers REMOTE: 206.221.180.138:7777)
# Revision
e8f2aca7d2bb16060dc630cd58459aee11fd1591

# Description
Powered crossbows can be charged while being on belt/back slot
# Steps to Reproduce
Buy/Steal/Make/Obtain powered crossbow.
Slot a rod in it.
Use it in-hand to start drawing back the string.
Place the crossbow on your belt/back slot.
You continue charging it there.
Result: you get a fully charged OP weapon at your back/belt, which you can then take from there no problem, ready to obliterate whoever stands in your way with 2 clicks.
# What you Expected
Nothing really.
# What Actually Happened
Squidmoko gave me a bunch of cookies for reporting this bug.

**Comment by Servalfang** [comment]
do powered crossbows do that much damage?

**Comment by kilozombie** [comment]
> do powered crossbows do that much damage?

When provided with a power cell, they can do over 50 brute damage. Doesn't sound like much? Remember that that's enough to break several bones and cause internal bleeding

**Comment by anunknownentity03** [comment]
balanced by the long charge up time from what i remember

**Comment by anunknownentity03** [comment]
and i think you need to stand still too while you pull back the string

**Comment by SonixApache** [comment]
I once found a crossbow and insta-killed the HoS with it without loading a battery. They can be pretty crazy.

**Comment by Armadingus** [comment]
They do 90 brute with a cell and fully drawn.
That's the reason why they are not craftable anymore.

**Comment by Killette3** [comment]
and everyone pretends pneumatic cannon does not exist

**Comment by drawsstuff** [comment]
Crossbows, when used correctly, are perhaps one of the most busted weapons; surpassing even shotguns slugs but being even **more** unwieldy to use.

-There's no way to craft, order, make more of them outside of space exploration.
-They take several seconds to charge up, from what I remember, around 15-20 seconds in-between shots.
-They don't reach as far as the traditional slug does, nor do they penetrate windows.
-They require you to keep them in hand when loaded, as dropping them, putting them in your backpack, ect will unload them / destroy the ammo currently loaded if a battery is slotted.

The issue with this bug stems from the last point: putting them on your back or belt and keeping them loaded / charging is a huge oversight as it directly bypasses several of the balance features in place designed to keep this weapon from being all-dominating.

**Comment by kilozombie** [comment]
I agree fully but it's funny pulling out your Murder Fucker from your belt and turning some poor unsuspecting person horizontal in 0.2 seconds
